### PR TITLE
feat(Prices): new field to allow users to input the quantity bought

### PR DESCRIPTION
**Pull request by raphodn**
### What

See #499

New Price field (specific to RECEIPT): `receipt_quantity`

Add validation, and API setup (read & write)

### CODE REVIEW

File: open_prices/prices/models.py [hunk 1]
```diff
@@ -361,6 +369,16 @@ def clean(self, *args, **kwargs):
                                     "proof",
                                     f"Proof {PROOF_FIELD} ({proof_field_value}) does not match the price {PROOF_FIELD} ({price_field_value})",
                                 )
+                if proof.type == proof_constants.TYPE_RECEIPT:
```
Comment (raphodn) [inline open_prices/prices/models.py:372]: what about GDPR proof types ?

Comment (raphodn) [reply open_prices/prices/models.py]: fixed here : https://github.com/openfoodfacts/open-prices/pull/498/commits/0e00b8aa1fbc6e336e8e9f2e6b4aba473b980464

File: open_prices/prices/models.py [hunk 2]
```diff
@@ -133,6 +134,13 @@ class Price(models.Model):
         related_name="prices",
     )
 
+    receipt_quantity = models.PositiveBigIntegerField(
```
Comment (TTalex) [inline open_prices/prices/models.py:137]: I wouldn't expect this value to reach BigInteger levels.
`PositiveSmallIntegerField` is probably enough (<32767)

Comment (raphodn) [reply open_prices/prices/models.py:137]: nice catch ! I think I went a bit fast and copy/pasted the setup of the `osm_id` field. will change to `PositiveIntegerField`

**Pull request merged**